<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(29, 5, 178, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>640</y>
      <width>170</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="friendListWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>341</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_friend_button">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>640</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nick_input">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>640</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>710</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="chat_line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>640</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_friend_button">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>710</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>139</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="chatLayout">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>709</width>
       <height>449</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
